keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -485,10 +486,11 @@ type ClusterDefaultNetworkPolicyIngress/EgressRule struct {
 }
 
 type ClusterDefaultNetworkPolicyPeer struct {
-	PodSelector  *metav1.LabelSelector
-	// required if a PodSelector is specified
-	Namespaces   *Namespaces
-	IPBlock      *networkingv1.IPBlock
+	PodSelector       *metav1.LabelSelector
+	// one of NamespaceSelector or Namespaces is required, if a PodSelector is specified
+	NamespaceSelector *metav1.LabelSelector
+	Namespaces        *Namespaces
```

Comment:
also need to mention that `NamepsaceSelector` and `Namespaces` are mutually exclusive

Reply:
Done in the latest commit